client/src/App.js

```diff
@@ -1,12 +1,99 @@
-import React from 'react';
+import React, { Component }from 'react';
 import './App.css';
+import { Router } from '@reach/router'
+import { Link } from '@reach/router'
+import NavBar from './Components/NavBar'
+import Home from './Components/Home'
+import About from './Components/About'
+import Projects from './Components/Projects'
+import Hobbies from './Components/Hobbies'
+import Work from './Components/Work'
+import Contact from './Components/Contact'
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
-  );
+
+// this changes the color of the navigation bar based on which page the user is on
+export default class App extends Component {
+  constructor(props) {
+    super(props) 
+
+    this.state = {
+
+      location: ""
+
+    }
+
+  }
+
+
+locationHandler = (evt) => {
+  this.setState({
+
+    location: evt.target.textContent
+
+  })
+}
+componentDidMount() {
+  window.scrollTo(0,0)
 }
```

Comment:
This is saying to scroll to the top of the page, but when we navigate to there, we're already at the top of the page